kubernetes/README.md

```diff
@@ -44,22 +45,35 @@ eksctl create cluster \
     --nodes=2
 ```
 
-Install Nvidia device plugin to enable GPU support on your cluster.
+2. Use `az aks` to create a Microsoft AKS cluster. This process takes ~4 minutes.
+
+```shell
+az aks create
+    --resource-group myResourceGroup \
+    --name mytorchelastic \
+    --node-vm-size Standard_NC6 \
+    --node-count 3 \
+    --location westus2 \
+    --kubernetes-version 1.15
+    --generate-ssh-keys
+```
+
+#### Install Nvidia device plugin to enable GPU support on your cluster.
 Deploy the following Daemonset:
 
 ```shell
 kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/1.0.0-beta4/nvidia-device-plugin.yml
```

Comment:
to double check, is this necessary for AKS?

Reply:
Yes, It's necessary for all K8s with Nvidia GPU